eng/pipelines/runtime-official.yml

```diff
@@ -122,6 +122,37 @@ stages:
         extraStepsParameters:
           name: MonoRuntimePacks
 
+  # Build Mono AOT offset headers once, for consumption elsewhere
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/mono/templates/generate-offsets.yml
+      buildConfig: release
+      platforms:
+      - Android_x64
```

Comment:
maybe add a comment here that this will actually generate offsets for all platforms, not just x64.